wiki/Music_theory/Tempo/en.md

```diff
@@ -0,0 +1,9 @@
+---
+stub: true
+---
+
+# Tempo
+
+**Tempo** is the speed at which a piece of music is played and is communicated, measured in beats per minute (BPM).
+
+[Screenshot of BPM on the beatmap information page](img/BPM_beatmap_info.png "The BPM of a song is shown on a beatmap's information page next to the metronome symbol")
```

Comment:
i don't think this screenshot is really necessary - i'd rather expand the lead paragraph a bit instead